spec/system/reward_buying_system_spec.rb

```diff
@@ -56,6 +56,19 @@
     expect(page).to have_text(reward.price)
     expect(reward.price) == 1
   end
+
+  it 'enables Admin to see list of rewards bought by the employee' do
+    create(:kudo, receiver_of_kudo: employee)
+    sign_in(employee)
+    visit(rewards_path)
+    find('tr', text: reward.title).click_on('Buy')
+    sign_in(admin_user)
```

Comment:
I would get rid of that part - you have the reward purchase already covered in the other test, and simply use a factory for reward, and an employee